### PR TITLE
feat: clear identity

### LINKED ISSUE #6188: Add option to clear identity
### Is your feature request related to a problem?

When setting identities, we could also offer the possibility to clear that identity and getting the deposit back. 
would not make a lot of sense for some other chains, unless their token goes really up in value, but for Polkadot, the current deposit is around $100.

### Describe the solution you would like

- A button inside the identity page allowing the person to clear their identity.
- fields should be pre-filled with their existent identity.
- locked previous identity deposit could be shown

more info: 
https://support.polkadot.network/support/solutions/articles/65000181981#Clearing-an-Identity

### Describe alternatives you have considered

using polkadot.js

### Screenshots

_No response_

### Are there opened related issues?

_No response_

### LINKED ISSUE #7328: Identity clear
image

- from what I understand, we don't have this yet in the UI 
- or is it when you remove all the fields and click create identity?

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

  👇 __ Let's make a quick check before the contribution.

  ## PR Type

  - [x] Bugfix
  - [x] Feature

  ## Needs Design check

  - @exezbcz @prury  please review
 

  ## Context

  - [x] Closes #7328
  - [x] Closes #6188
  - [ ] Requires deployment <snek/rubick/worker>

  #### Did your issue had any of the "$" label on it?

  - [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)

  ## Screenshot 📸

  - [x] My fix has changed UI
image

  ## Copilot Summary
  
### <samp>🤖 Generated by Copilot at c147be4</samp>

Improved the identity form component and the useIdentity composable to handle different chains and identity data. Added new translations and styles to support the UI changes. Updated `identity.set` translation key and added `identity.clear`, `identity.existing`, and `identity.establishedIdentity`. Added `.border-k-shade` class to `global.scss`.

  
### <samp>🤖 Generated by Copilot at c147be4</samp>

> _Sing, O Muse, of the mighty pull request_
> _That changed the identity form with skill and zest_
> _How the coder, wise in translation keys, updated the set_
> _And added new ones for clear, existing, and establishedIdentity yet_

### CODE REVIEW

**Comment by prury** [comment]
identity burning works and value previously deposited gets refunded

**Comment by exezbcz** [comment]
this button should have the same effect as the ones in profile sidebar 
image
image

otherwise okay for me, thnkas!

**Comment by Jarsen136** [comment]
> this button should have the same effect as the ones in profile sidebar image image
> 
> otherwise okay for me, thnkas!

✅ Done @exezbcz 
image

File: components/common/IdentityForm.vue [hunk 1]
```diff
@@ -147,23 +179,29 @@ const enhanceIdentityData = (): Record<string, any> => {
...
+      refetchIdentity()
+    }
+    showNotification(
+      isDelete
+        ? `[Identity] You have cleared your account's identity since block ${block}`
+        : `[Identity] You are known as ${identity.value.display} since block ${block}`,
+      notificationTypes.success
+    )
+  })
 }
```
Comment (vikiival) [inline components/common/IdentityForm.vue:205]: I would make it as a two separated functions

**Review by vikiival — COMMENTED** [review]
Otherwise ok

File: components/common/IdentityForm.vue (hunk 1 above)
Comment (Jarsen136) [reply components/common/IdentityForm.vue]: done

File: styles/global.scss [hunk 2]
```diff
@@ -321,7 +322,7 @@ a.has-text-grey {
 
 .border-k-shade {
   @include ktheme() {
-    border-color: theme('k-shade');
+    border-color: theme('k-shade');;
```
Comment (roiLeo) [inline styles/global.scss:325]:
```suggestion
    border-color: theme('k-shade');
```

**Review by roiLeo — APPROVED** [review]

**Comment by prury** [comment]
did a second round of tests after the changes and is working fine ✔️

**Review by vikiival — APPROVED** [review]

**Pull request merged**